wire-grpc-tests/src/test/java/com/squareup/wire/GrpcClientTest.kt

```diff
@@ -352,59 +355,110 @@ class GrpcClientTest {
   }
 
   @Test
-  fun duplexSuspend_errorsAreBubbledUpWhenSendingFailed() {
+  fun duplexSuspend_requestChannelThrowsWhenResponseChannelExhausted() {
```

Comment:
I think we need one more test, which is this but the server sends an error instead of SendCompleted at the end

Reply:
I added one.